src/components/CreateReview.js

```diff
@@ -0,0 +1,75 @@
+import React from "react";
+import { connect } from "react-redux";
+import { postReview } from "../actions/reviewActions";
+
+class CreateReview extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      title: "",
+      body: "",
+      productId: "",
+      userId: ""
+    };
+    this.onHandle = this.onHandle.bind(this);
```

Comment:
If you have babel configured correctly with `class fields` you can omit all this annoying binding garbage and do the following cool kid stuff:

```javascript
class CoolKid extends Component {
  state = { cool: true }

  onSubmit = () => {
    // already lexically bound to this
  }
}
```